src/variables.jl

```diff
@@ -55,6 +88,59 @@ function add_constrained_variable(model::ModelLike, set::AbstractScalarSet)
     return variable, constraint
 end
 
+"""
+    supports_add_constrained_variables(
+        model::ModelLike,
+        S::Type{<:AbstractScalarSet}
+    )::Bool
+
+Return a `Bool` indicating whether `model` supports constraining a vector of
+variables to belong to a set of type `S` either on creation of the vector of
+variables with [`add_constrained_variables`](@ref) or after the variable is
+created with [`add_constraint`](@ref).
+
+By default, if `S` is `Reals` then this function returns `true` and otherwise,
+it falls back to `supports_add_constrained_variables(model, Reals) &&
+supports_constraint(model, MOI.VectorOfVariables, S)` which is the correct
+definition for most models.
+
+## Example
+
+In the standard conic form, the variables are grouped into several cones
```

Comment:
Which standard conic form is this? A reference or link to an example (e.g., SCS) might help.